### PR TITLE
ss/dcos-54050 Adding section on creating secrets from a file via DC/O…

### PR DESCRIPTION
**Pull request by SarahStegall**
## Jira Ticket
https://jira.mesosphere.com/browse/DCOS-54050

## Description of changes being made

Created and added new screen images from Secrets UI. 
Added procedure from https://docs.google.com/document/d/1Xa197A0H8gm-X25VGEpvdCeQvj6wyA1UEewsCeWML2k/edit?usp=sharing
Tested with local build

## Versions affected
1.13
1.14

### CODE REVIEW

**Comment by SarahStegall** [comment]
Also requested review from Dima Rozhkov.

File: pages/1.13/security/ent/secrets/create-secrets/index.md [hunk 1]
```diff
@@ -97,4 +97,38 @@ The contents of the file (referred to below as `my-secret.txt`) can be any text
   dcos security secrets create -f my-secret.txt developer/my-secret
   ```
 
-  <p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.
+  <p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.</p>
+
+# Creating secrets from a file via the DC/OS UI
```
Comment (TattdCodeMonkey) [inline pages/1.13/security/ent/secrets/create-secrets/index.md]: We should also update the screenshots for the `Creating key-value pair secrets using the UI` in this file. Specifically the modal `png` on line 51 is now out of date.

File: pages/1.13/gui/secrets/index.md [hunk 2]
```diff
@@ -10,6 +10,66 @@ model: /1.13/data.yml
...
+1. Type or paste the secret into the **Value** box.
+
+    ![Secret ID/Value Fields](/1.13/img/create-secret.png)
+
+    Figure 3. Creating a new Secret
+
+1. Click **Create**.
+
+# Creating secrets from a file 
+
```
Comment (TattdCodeMonkey) [inline pages/1.13/gui/secrets/index.md]: Should we include the maximum size warning here too? We have it in the `pages/1.13/security/ent/secrets/create-secrets/index.md` file.

File: pages/1.13/security/ent/secrets/create-secrets/index.md (hunk 1 above)
Comment (SarahStegall) [reply pages/1.13/security/ent/secrets/create-secrets/index.md]: Done.

File: pages/1.13/gui/secrets/index.md (hunk 2 above)
Comment (SarahStegall) [reply pages/1.13/gui/secrets/index.md]: Just to be clear, are you talking about this warning?

`IMPORTANT: The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.`

Comment (TattdCodeMonkey) [reply pages/1.13/gui/secrets/index.md]: That’s the one!

**Comment by SarahStegall** [comment]
All changes tested and passed in local build.

**Pull request merged**